### PR TITLE
Hot fix infection feedback term description in `model_definition`

### PR DESCRIPTION
**Pull request by kaitejohnson**
I'm not sure how I missed this, we totally missed updating the `model_definition.md` when we changed the infection feedback prior. 

I just edited it to reflect the fact that we fit the model across the 2023-24 epidemic peak, and averaged the posterior estimates of the infection feedback term across a few jurisdictions to get a reasonable prior for this. https://github.com/CDCgov/wastewater-informed-covid-forecasting/pull/197

### CODE REVIEW

File: model_definition.md [hunk 1]
```diff
@@ -265,11 +265,11 @@ We use informative priors for parameters that have been well characterized in th
 | Total genomes shed per infected individual | $\log_{10}[G] \sim \mathrm{Normal}(9, 2)$ | Watson et al 2023[^Watson]   |
 | Initial infections per capita $I_0$ | $I_0 \sim \mathrm{Beta}(1 + k i_\mathrm{est}, 1 + k (1-i_\mathrm{est}))$ | where $i_\mathrm{est}$ is the sum of the last 7 days of hospital admissions, divided by jurisdiction population, and divided by the prior mode for $p_\mathrm{hosp}$, and $k = 5$ is a parameter governing the informativeness ("certainty") of the Beta distribution |
 | Initial exponential growth rate | $r \sim \mathrm{Normal}(0, 0.01)$ | Chosen to assume flat dynamics prior to observations |
-| Infection feedback term | $\gamma \sim \mathrm{logNormal}(6.37, 0.4)$ | Weakly informative prior chosen to have a mode of 500 in natural scale, based on posterior estimates of peaks from prior seasons in a few jurisdictions |
+| Infection feedback term | $\gamma \sim \mathrm{logNormal}(4.50, 0.64)$ | Weakly informative prior chosen to have a mode of 90 in natural scale, based on posterior estimates of the infection feedback fit to the peak in the 2023-24 epidemic season in a few jurisdictions |
```
Comment (dylanhmorris) [inline model_definition.md:268]: 1. Would be ideal to provide the exact number of jurisdictions if you have it.
2. "To the season" should perhaps be clarified if possible. IIRC you used a longer fitting window than the 90 day production window. Maybe just say the dates?

```suggestion
| Infection feedback term | $\gamma \sim \mathrm{logNormal}(4.50, 0.64)$ | Weakly informative prior chosen to have a mode of 90 in natural scale, based on posterior estimates of the infection feedback from model fits to the 2023-24 epidemic season in a few jurisdictions |
```

**Review by dylanhmorris — APPROVED** [review]
A couple minor comments, not critical

File: model_definition.md (hunk 1 above)
Comment (kaitejohnson) [reply model_definition.md]: Added the 5 states, it was multiple forecast dates (March 4th and March 11th) going back 365 days each.

File: model_definition.md [hunk 2]
```diff
@@ -265,11 +265,11 @@ We use informative priors for parameters that have been well characterized in th
...
 | Modal site-level observation standard deviation | $\hat{\sigma}\_c \sim \mathrm{Normal}(1,1)$ | Weakly informative prior chosen to allow the mode to be either small or large |
-| Standard deviation of the Normal distribution of individual log observation standard deviations $\log(\sigma\_{cij})$ (site-lab combination specific, with an inferred modal s.d. $\hat{\sigma}\_c$) | $\sigma\_{\log \sigma\_c} \sim \mathrm{Normal}(0, \log(2))$ | Weakly informative prior which allows for individual s.d.s to be either clustered around the mode or more dispersed |
+| Standard deviation of the Normal distribution of individual log observation standard deviations $\log(\sigma\_{cij})$ (site-lab combination specific, with an inferred modal standard deviation $\hat{\sigma}\_c$) | $\sigma\_{\log \sigma\_c} \sim \mathrm{Normal}(0, \log(2))$ | Weakly informative prior which allows for individual standard deviationss to be either clustered around the mode or more dispersed |
```
Comment (dylanhmorris) [inline model_definition.md:272]:
```suggestion
| Standard deviation of the Normal distribution of individual log observation standard deviations $\log(\sigma\_{cij})$ (site-lab combination specific, with an inferred modal standard deviation $\hat{\sigma}\_c$) | $\sigma\_{\log \sigma\_c} \sim \mathrm{Normal}(0, \log(2))$ | Weakly informative prior which allows for individual standard deviations to be either clustered around the modal standard deviation or more dispersed |
```

**Review by dylanhmorris — COMMENTED** [review]
Caught one typo.

**Review by dylanhmorris — APPROVED** [review]

**Pull request merged**